client/src/components/PublicPages/AuditPlanner/AuditPlanCard.tsx

```diff
@@ -0,0 +1,156 @@
+  // Scroll the card, specifically the sample size, into view when it first appears
+  const sampleSizeSectionRef = useRef<HTMLDivElement>(null)
+  useEffect(() => {
+    if (sampleSizeSectionRef.current) {
+      sampleSizeSectionRef.current.scrollIntoView({ behavior: 'smooth' })
+    }
+  }, [])
+
+  const [selectedAuditType, setSelectedAuditType] = useState<
+    Exclude<AuditType, 'HYBRID'>
+  >('BALLOT_POLLING')
+  const [riskLimitPercentage, setRiskLimitPercentage] = useState(
+    DEFAULT_RISK_LIMIT_PERCENTAGE
+  )
+  const [
+    debouncedRiskLimitPercentage,
+    setDebouncedRiskLimitPercentage,
+  ] = useState(riskLimitPercentage)
+  const sampleSizes = useSampleSizes(electionResults, {
+    // We display an inline error message instead
+    showToastOnError: false,
+  })
+
+  return (
+    <Container data-testid="auditPlanCard" disabled={disabled} elevation={1}>
+      <InnerContainer>
+        <Heading>Audit Plan</Heading>
+
+        <Section>
+          <SubHeading id="auditMethodLabel">Audit Method</SubHeading>
+          <SegmentedControl
+            aria-labelledby="auditMethodLabel"
+            disabled={disabled}
+            fill
+            large
+            onChange={setSelectedAuditType}
+            options={[
+              { label: 'Ballot Polling', value: 'BALLOT_POLLING' },
+              { label: 'Ballot Comparison', value: 'BALLOT_COMPARISON' },
+              { label: 'Batch Comparison', value: 'BATCH_COMPARISON' },
+            ]}
+            value={selectedAuditType}
+          />
+        </Section>
+
+        <Section>
+          <SubHeading>Risk Limit</SubHeading>
+          <Slider
+            disabled={disabled}
+            labelRenderer={value => (
+              <span
+                // A hack to display the listed values on both the slider axis and the slider label,
+                // and all other values on only the slider label. More recent versions of the
+                // Blueprint slider actually support this differentiation
+                style={{
+                  color: [0, 5, 10, 15, 20].includes(value)
+                    ? undefined
+                    : 'white',
+                }}
+              >
+                {value}%
+              </span>
+            )}
+            min={0}
+            max={20}
+            onChange={setRiskLimitPercentage}
+            onRelease={setDebouncedRiskLimitPercentage}
+            value={riskLimitPercentage}
+          />
+        </Section>
+      </InnerContainer>
+
+      <SampleSizeSection ref={sampleSizeSectionRef}>
+        <SubHeading>Estimated Sample Size</SubHeading>
+        <SampleSize
+          auditType={selectedAuditType}
+          disabled={disabled}
+          error={sampleSizes.error || undefined}
+          isComputing={sampleSizes.isFetching}
+          sampleSize={
+            sampleSizes.data?.[selectedAuditType][
+              debouncedRiskLimitPercentage.toString()
+            ] || 0
```

Comment:
what does the `|| 0` case account for here?

Reply:
Good q - just accounting for the case that `sampleSizes.data` is undefined. In practice, whenever `sampleSizes.data` is undefined, the `SampleSize` component will be in some other state (e.g. disabled, error, or loading)

That said, I tweaked the logic just a bit for clarity since `|| 0` is a bit misleading: https://github.com/votingworks/arlo/pull/1597/commits/ea613e1ef71cbe8e24a6305e66c3d6ca065444fa

Reply:
^ Actually made a tiny mistake with https://github.com/votingworks/arlo/commit/ea613e1ef71cbe8e24a6305e66c3d6ca065444fa 🙈

The `sampleSize === undefined` condition should come after the loading spinner condition. It's now preventing the loading spinner from displaying. Will fix in one of my other in-flight PRs

Reply:
https://github.com/votingworks/arlo/pull/1599/commits/0dc6bc9c5f060fd3157866152af865e51061666a